components/esp_modem/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ else()
     set(platform_srcs src/esp_modem_primitives_freertos.cpp
                       src/esp_modem_api_target.cpp
                       src/esp_modem_uart.cpp
+                      src/esp_modem_uart_dma.cpp
                       src/esp_modem_term_uart.cpp
                       src/esp_modem_netif.cpp)
-    set(dependencies driver esp_event esp_netif)
+    set(dependencies driver esp_event esp_netif esp_driver_uart)
```

Comment:
@david-cermak Is there any ETA for this change?

Currently it breaks our builds with esp-idf/master. (We build `esp_modem_usb_dte` in our USB CI, which in turn pulls in `esp_modem`. https://github.com/espressif/esp-usb/actions/runs/18191462194/job/51787270030?pr=276#step:4:1917

We can temporarily disable the build job if next esp_modem needs more time

Reply:
> We can temporarily disable the build job if next esp_modem release needs more time

Please disable the build for now, got some breaking changes on master already, so I'll have to branch off...